python/tests/conftest.py

```diff
@@ -1259,3 +1276,22 @@ def fixture_glue(_aws_credentials: None) -> Generator[boto3.client, None, None]:
     """Mocked glue client"""
     with mock_glue():
         yield boto3.client("glue", region_name="us-east-1")
+
+
+@pytest.fixture
+def adlfs_fsspec_fileio(request: pytest.FixtureRequest) -> Generator[FsspecFileIO, None, None]:
+    from azure.storage.blob import BlobServiceClient
```

Comment:
mypy is complaining about this import, not sure how to make it work since adlfs is an optional dependency.

Reply:
We add excludes to the pyproject.toml file. You'll see a huge list.